source/backend/api/Services/CompensationRequisitionService.cs

```diff
@@ -107,27 +93,50 @@ public bool DeleteCompensation(long compensationId)
 
             var currentCompensation = _compensationRequisitionRepository.GetById(compensationId);
 
-           /* TODO: Fix compensation requisition
-            var currentAcquisitionStatus = GetCurrentAcquisitionStatus(currentCompensation.AcquisitionFileId);
-
-            if (!_statusSolver.CanEditOrDeleteCompensation(currentAcquisitionStatus, currentCompensation.IsDraft) && !_user.HasPermission(Permissions.SystemAdmin))
+            if (currentCompensation.AcquisitionFileId is not null)
             {
-                throw new BusinessRuleViolationException("The file you are editing is not active or hold, so you cannot save changes. Refresh your browser to see file state.");
+                var currentAcquisitionStatus = GetCurrentAcquisitionStatus((long)currentCompensation.AcquisitionFileId);
+
+                if (!_acquisitionStatusSolver.CanEditOrDeleteCompensation(currentAcquisitionStatus, currentCompensation.IsDraft) && !_user.HasPermission(Permissions.SystemAdmin))
```

Comment:
Hmm, does something similar apply to leases?

Reply:
I asked 'Dev' and his answer was 'not at this moment'.